src/golomb.h

```diff
@@ -96,40 +97,39 @@ class encoder
     int         encoded_count;
 
 public:
-    encoder( OutputIt output, size_t k = 0u ) noexcept
-        : output( output )
-        , k( k )
-        , base( 1u << k )
-        , zeros_threshold( 1 + k )
+    encoder( OutputIt output_in, size_t k_in = 0u ) noexcept
```

Comment:
Today I learned... Initialiser list doesn't conflict with input parameters as far as names are concerned.
I was worried about the ambiguity of the names, since the members have the exact same name as the parameters.